### ISSUE TITLE
Update npm package

### ISSUE THREAD

**Issue by simplePCBuilding** [impress/impress.js#838]
The last version of impress.js for npm is 1.1.0. Current release is 2.0.0. I suggest updating to 2.0.0 (would also be willing to now maintain impress.js npm package) and add the new readme I have created in PR

**Comment by simplePCBuilding** [comment]
I will also add a direct cdn download link for the main js file.

**Comment by simplePCBuilding** [comment]
I have created the readme and opened a PR
https://github.com/impress/impress.js/pull/836_

**Comment by p3palazzo** [comment]
Any news on this?

**Comment by simplePCBuilding** [comment]
Not really. I am also working on a new version for impress, so we'll probably push for V3 and V2 at the same time
